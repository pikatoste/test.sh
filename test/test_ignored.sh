test_01() {
  set_test_name "test_01"
  false
}

test_02() {
  set_test_name "test_02"
  true
}

source "$(dirname "$(readlink -f "$BASH_SOURCE")")"/../test.sh

! subshell "FAIL_FAST=1 SUBSHELL=always run_tests"
# TODO: verify that the [ignored] line is printed
