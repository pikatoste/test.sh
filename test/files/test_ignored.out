[0;31m* test_01[0m
[0;34m* [skipped] test_02[0m
